substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/classinitialization/ClassInitializationSupport.java

```diff
@@ -187,18 +187,7 @@ InitKind ensureClassInitialized(Class<?> clazz, boolean allowErrors) {
     }
 
     private InitKind reportInitializationError(String msg, Class<?> clazz, Throwable t) {
-        if (unsupportedFeatures != null) {
-            /*
-             * Report an unsupported feature during static analysis, so that we can collect multiple
-             * error messages without aborting analysis immediately. Returning InitKind.RUN_TIME
-             * ensures that analysis can continue, even though eventually an error is reported (so
-             * no image will be created).
-             */
-            unsupportedFeatures.addMessage(clazz.getTypeName(), null, msg, null, t);
```

Comment:
You can actually remove the field `unsupportedFeatures` and all the code that sets it. And inline away this method.